evm_loader/program/src/account_storage/apply.rs

```diff
@@ -49,16 +52,28 @@ impl<'a> ProgramAccountStorage<'a> {
         system_program: &program::System<'a>,
         operator: &Operator<'a>,
         actions: Vec<Action>,
-    ) -> Result<(), ProgramError> {
-
+        accounts_operations: Option<AccountsOperations>,

```

Comment:
Why do you need `Option` here? You can use empty vector and simplify you code.

Reply:
Done.